packages/sitecore-jss-nextjs/src/services/graphql-sitemap-service.test.ts

```diff
@@ -318,6 +333,42 @@ describe('GraphQLSitemapService', () => {
       expect(sitemap).to.deep.equal(sitemapServiceResult);
     });
 
+    it('should use a jssTemplateId, if provided', async () => {
+      const jssAppTemplateId = generateGUID();
+      // mock a random template id
+      const randomId = generateGUID();
```

Comment:
I'm not sure that it makes sense to generate random uid, anyway we are not using real services, basically you can hardcode some id.